src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -353,40 +347,37 @@ void testGetFunctions() throws SQLException {
                     "FUNCTION_TYPE",
                     "SPECIFIC_NAME",
                 };
-        for (int i = 0; i < columns.length; ++i) {
-            assertEquals(rsmd.getColumnName(i + 1), columns[i]);
-            assertEquals(rsmd.getColumnLabel(i + 1), columns[i]);
-        }
-        assertEquals(countRows(rs), 117);
+
+        ResultSet rs = databaseMetaData.getFunctions(null, null, null);
```

Comment:
This is where I'd add databaseMetaData.getFunctions(null, null, "%") and make sure it also return 117 rows for example.